diff --git b/src/ee9.adb a/src/ee9.adb
index ec2da14..4d34d4f 100644
--- b/src/ee9.adb
+++ a/src/ee9.adb
@@ -26,6 +26,7 @@ with IOC.equipment;
 with IOC.slow.shift.TP;
 with IOC.slow.shift.TR;
 with settings;
+with get_runtime_paths;
 
 with execute;
 with say_goodbye;
@@ -42,7 +43,7 @@ procedure ee9 is
 
    package CLI renames Ada.Command_Line;
 
-   the_log_file_name : constant String := "KDF9_log.txt";
+   the_log_file_name : constant String := get_runtime_paths & "logs/KDF9.log";
 
    a_command_line_error_is_detected : exception;
 
diff --git b/src/host_io.adb a/src/host_io.adb
index 14059bb..57781b5 100644
--- b/src/host_io.adb
+++ a/src/host_io.adb
@@ -17,6 +17,7 @@
 with Ada.Characters.Latin_1;
 --
 with OS_specifics;
+with get_runtime_paths;
 
 use  Ada.Characters.Latin_1;
 --
@@ -50,7 +51,7 @@ package body host_IO is
                    mode       : in POSIX.access_mode) is
       fd : Integer;
    begin -- open
-      fd := POSIX.open(file_name, mode);
+      fd := POSIX.open(get_runtime_paths & file_name, mode);
       open(the_stream, file_name, mode, fd);
    exception
       when POSIX_IO_error =>
diff --git a/src/ioc-slow-shift-fw.adb b/src/ioc-slow-shift-fw.adb
index a6ef6f5..88dab17 100644
--- a/src/ioc-slow-shift-fw.adb
+++ b/src/ioc-slow-shift-fw.adb
@@ -78,7 +78,7 @@ package body IOC.slow.shift.FW is
    end escaped;
 
    red_font   : constant String := escaped("RED_FONT",   "ESC[0mESC[31m");
-   black_font : constant String := escaped("BLACK_FONT", "ESC[0mESC[30m");
+   black_font : constant String := escaped("BLACK_FONT", "ESC[0m");
    underline  : constant String := escaped("UNDERLINE",  "ESC[4m");
    plain_font : constant String := escaped("PLAIN_FONT", "ESC[0m");
 
