diff --git a/kal3/kal3.c b/kal3/kal3.c
index c01618a..74aaf42 100644
--- a/kal3/kal3.c
+++ b/kal3/kal3.c
@@ -373,9 +373,9 @@ void ptwrite(int dv, int start, int stop)
 void ptbinout()
 // output a KDF9 paper tape binary program
 // not yet complete -- needs A-block and program identifier
-{  int fout = open("a.out", O_CREAT + O_TRUNC + O_WRONLY + O_BINARY, 0640);
+{  int fout = open("/tmp/a.out", O_CREAT + O_TRUNC + O_WRONLY + O_BINARY, 0640);
    if  ( fout < 0 )
-      perror("a.out");
+      perror("/tmp/a.out");
    else
    {  if  ( ablk != 0 )        // A-block generation not suppressed to satisfy ee9 -- temp ??
       {  buff[16] = ptchar[075];                  // end-message
@@ -400,11 +400,11 @@ void ptbinout()
 void mtbinout()
 // output a KDF9 mag tape B-block and C-block
 // must be called after ptbinout()
-{  int fout = open("mt.out", O_CREAT + O_TRUNC + O_WRONLY + O_BINARY, 0640);
+{  int fout = open("/tmp/mt.out", O_CREAT + O_TRUNC + O_WRONLY + O_BINARY, 0640);
    int i;
    int prn[4];         // program name in KDF9 chars
    if  ( fout < 0 )
-      perror("mt.out");
+      perror("/tmp/mt.out");
    else
    {  prn[0] = 07025502;        // load up words 2 and 3
       prn[1] = (progname[0]<<18) + (progname[1]<<12) + (progname[2]<<6) + progname[3];
